custom-fonts.md

```diff
@@ -23,6 +41,72 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Using the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+as a constant. 
+Then set the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html) `fontFamily` property 
+and use the font family specified in the `pubsec.yaml` delcarations. 
+The font family `pubsec.yaml` declaration will look something like `family: [declared-font-family]`.
```

Comment:
is this `family:` something the user needs to specify? If so, please add this to the pubspec.yaml example above. Then you can rewrite this sentence: "The fontFamily name must match the `family:` declaration in the pubspec.yaml"